.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is for VS Code only. Tells VS Code to open the project in a Docker container and configures it. Ignored otherwise. See the readme.